src/server/pachyderm_test.go

```diff
@@ -9347,6 +9346,8 @@ func TestPipelineAutoscaling(t *testing.T) {
 }
 
 func TestListDeletedDatums(t *testing.T) {
+	// TODO(2.0 optional): Duplicate file paths from different datums no longer allowed.
+	t.Skip("Duplicate file paths from different datums no longer allowed.")
```

Comment:
It seems like this test will never pass if we changed the behavior?  Should we delete it?

Reply:
It just needs to be updated. It seems like a decent enough test for joins.